cirq-core/cirq/transformers/routing/route_circuit_cqc.py

```diff
@@ -245,17 +245,32 @@ def _get_one_and_two_qubit_ops_as_timesteps(
         The i'th entry in the nested two-qubit and single-qubit ops correspond to the two-qubit
         gates and single-qubit gates of the i'th timesteps respectively. When constructing the
         output routed circuit, single-qubit operations are inserted before two-qubit operations.
+
+        Raises:
+            ValueError: if circuit has intermediate measurement op's that act on three or more qubits and result is stored.
```

Comment:
`ValueError: if circuit has intermediate measurements that act on three or more qubits with a custom key.`